frontend/packages/dev-console/src/components/import/GitImportForm.tsx

```diff
@@ -61,13 +61,14 @@ const GitImportForm: React.FC<FormikProps<FormikValues> & GitImportFormProps> =
               project={values.project}
               noProjectsAvailable={projects.loaded && _.isEmpty(projects.data)}
             />
-            {values.import.selectedStrategy.type !== ImportStrategy.DEVFILE && (
-              <>
-                <ResourceSection />
-                <PipelineSection builderImages={builderImages} />
-                <AdvancedSection values={values} />
-              </>
-            )}
+            {importType !== 'devfile' &&
```

Comment:
Please use the ImportTypes enum. Please also replace the other 'devfile' below.

```suggestion
            {importType !== ImportTypes.devfile &&
```